src/main.rs

```diff
@@ -257,6 +257,10 @@ struct CheckRelease {
     )]
     all_features: bool,
 
+    /// Which target to build the crate for, to check platform-specific APIs.
+    #[arg(long)]
+    target: Option<String>,
```

Comment:
Thoughts on calling this field `build_target` instead (+updating all other internal names to `build_target`), then making an alias for `--target` to match cargo?

Cargo seems split: it calls it `--target`, `build.target`, and `CARGO_BUILD_TARGET`. I'm inclined to support `--target` to match it in the CLI, but I feel like internally and via the lib API `build_target` is going to be a less confusing name. For example, we don't want it to be confused with a custom target directory, which one day might also be another `Option<String>` field with `target` in the name.

Reply:
It might also be nice to add an example of a build target in the doc comment, say e.g. `x86_64-unknown-linux-gnu`